Makefile.am

```diff
@@ -41,13 +41,20 @@ DISTCLEANFILES = $(PACKAGE)-$(VERSION).tar.gz \
 	configure \
 	install-sh \
 	missing \
-	Makefile.in
+	Makefile.in \
+	.bash_history \
```

Comment:
should probably skip this

Reply:
All it will do is let `make distclean` or `yarn clean` be able to clean up extra `mock_runner.sh` generated files.  It is an entirely manual step.  I really only added them here so I could clean up my own project folder after running the `mock_runner.sh`.

Reply:
I've seen complaints on vdsm project about the same...it's annoying shortcoming of stdci.
it's fine with me...just make sure it dosn't wipe out your own user's bash history:)